temboardagent/plugins/monitoring/__init__.py

```diff
@@ -43,99 +44,102 @@
 
 
-@routes.get(b'/db_size', check_key=True)
+@routes.get(b'/probe/db_size', check_key=True)
 def get_probe_db_size(http_context, app):
     return api_run_probe(probe_db_size(app.config.monitoring), app.config)
 
 
-@routes.get(b'/tblspc_size', check_key=True)
+@routes.get(b'/probe/tblspc_size', check_key=True)
 def get_probe_tblspc_size(http_context, app):
     return api_run_probe(probe_tblspc_size(app.config.monitoring), app.config)
 
 
-@routes.get(b'/filesystems_size', check_key=True)
+@routes.get(b'/probe/filesystems_size', check_key=True)
 def get_probe_filesystems_size(http_context, app):
     return api_run_probe(probe_filesystems_size(app.config.monitoring),
                          app.config)
 
 
-@routes.get(b'/cpu', check_key=True)
+@routes.get(b'/probe/cpu', check_key=True)
 def get_probe_cpu(http_context, app):
     return api_run_probe(probe_cpu(app.config.monitoring), app.config)
 
 
-@routes.get(b'/process', check_key=True)
+@routes.get(b'/probe/process', check_key=True)
 def get_probe_process(http_context, app):
     return api_run_probe(probe_process(app.config.monitoring), app.config)
 
 
-@routes.get(b'/memory', check_key=True)
+@routes.get(b'/probe/memory', check_key=True)
 def get_probe_memory(http_context, app):
     return api_run_probe(probe_memory(app.config.monitoring), app.config)
 
 
-@routes.get(b'/loadavg', check_key=True)
+@routes.get(b'/probe/loadavg', check_key=True)
 def get_probe_loadavg(http_context, app):
     return api_run_probe(probe_loadavg(app.config.monitoring), app.config)
 
 
-@routes.get(b'/wal_files', check_key=True)
+@routes.get(b'/probe/wal_files', check_key=True)
 def get_probe_wal_files(http_context, app):
     return api_run_probe(probe_wal_files(app.config.monitoring), app.config)
 
 
-@routes.get(b'/replication_lag', check_key=True)
+@routes.get(b'/probe/replication_lag', check_key=True)
 def get_probe_replication_lag(http_context, app):
     return api_run_probe(probe_replication_lag(app.config.monitoring),
                          app.config)
 
 
-@routes.get(b'/temp_files_size_delta', check_key=True)
+@routes.get(b'/probe/temp_files_size_delta', check_key=True)
 def get_probe_temp_files_size_delta(http_context, app):
     return api_run_probe(probe_temp_files_size_delta(app.config.monitoring),
                          app.config)
 
 
-@routes.get(b'/replication_connection', check_key=True)
+@routes.get(b'/probe/replication_connection', check_key=True)
 def get_probe_replication_connection(http_context, app):
     return api_run_probe(probe_replication_connection(app.config.monitoring),
                          app.config)
 
 
-@routes.get(b'/heap_bloat', check_key=True)
+@routes.get(b'/probe/heap_bloat', check_key=True)
 def get_probe_heap_bloat(http_context, app):
     return api_run_probe(probe_heap_bloat(app.config.monitoring), app.config)
 
 
-@routes.get(b'/btree_bloat', check_key=True)
+@routes.get(b'/probe/btree_bloat', check_key=True)
```

Comment:
Can you explain (in the commit message, ideally) the motivation behind the new routing (move of everything into "/probe" and using "/" for the main metrics endpoint)? It's not quite obvious to me and might deserve some discussion.

Reply:
Commit message updated.